R2API/ScriptableObjects/ArtifactCode.cs

```diff
@@ -3,14 +3,16 @@
 using System.Security.Cryptography;
 using UnityEngine;
 
-namespace R2API.ScriptableObjects {
+namespace R2API {
```

Comment:
I assume this was needed for unity editor support

Reply:
yes. otherwise it doesnt work on the editor